packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -41,7 +41,7 @@ class StripeSdkModule(reactContext: ReactApplicationContext, cardFieldManager: S
   private var confirmPaymentClientSecret: String? = null
 
   private val mActivityEventListener = object : BaseActivityEventListener() {
-    override fun onActivityResult(activity: Activity, requestCode: Int, resultCode: Int, data: Intent) {
+    override fun onActivityResult(activity: Activity, requestCode: Int, resultCode: Int, data: Intent?) {
```

Comment:
are you sure that intent should be nullable?  I do not see this at the original implementation of react-native. https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/BaseActivityEventListener.java

@jonasbark  can you see some potential issues here?